src/Stratis.Bitcoin.Features.RPC/Stratis.Bitcoin.Features.RPC.csproj

```diff
@@ -35,7 +35,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\NBitcoin\NBitcoin.csproj" />
-    <ProjectReference Include="..\Stratis.Bitcoin.Features.MemoryPool\Stratis.Bitcoin.Features.MemoryPool.csproj" />
+    <ProjectReference Include="..\Stratis.Bitcoin.Features.Consensus\Stratis.Bitcoin.Features.Consensus.csproj" />
```

Comment:
This is still wrong but leave it for now. we can remove the dependency on consensus it in another PR.